src/crs.jl

```diff
@@ -122,15 +122,22 @@ mactype(C::Type{<:CRS}) = numtype(lentype(C))
 
 Checks whether or not `coords₁` and `coords₂` are approximately equal as
 if they were tuples, i.e., their coordinate values are compared one by one
-with `isapprox` and the forwarded `kwargs`.
+with `isapprox` and the forwarded `kwargs`, except for `rtol` and `atol`.
+The `rtol` and `atol` are fixed based on the coordinate types. For example,
+angular coordinates in radians are compared with `rtol=0` and `atol=√eps(2π)`.
 
 In the case of different CRS types, converts `coords₂` to the `typeof(coords₁)`,
 handling possibly different datums, units and machine types.
 """
 Base.isapprox(coords₁::CRS, coords₂::CRS; kwargs...) = isapprox(coords₁, convert(typeof(coords₁), coords₂); kwargs...)
 
-Base.isapprox(coords₁::C, coords₂::C; kwargs...) where {C<:CRS} =
-  all(ntuple(i -> isapprox(getfield(coords₁, i), getfield(coords₂, i); kwargs...), nfields(coords₁)))
+function Base.isapprox(coords₁::C, coords₂::C; kwargs...) where {C<:CRS}
+  all(1:nfields(coords₁)) do i
+    c₁ = getfield(coords₁, i)
+    c₂ = getfield(coords₂, i)
+    isapprox(c₁, c₂; rtol=rtol(c₁), atol=atol(c₁), kwargs...)
+  end
+end
```

Comment:
@juliohm in this implementation, it is not allowed to pass a custom `atol` and `rtol`. Was this the expected behavior?